src/rust/src/x509/verify.rs

```diff
@@ -214,7 +365,9 @@ self_cell::self_cell!(
 )]
 pub(crate) struct PyVerifiedClient {
     #[pyo3(get)]
-    subjects: pyo3::Py<pyo3::PyAny>,
+    subject: pyo3::Py<pyo3::PyAny>,
+    #[pyo3(get)]
+    sans: Option<pyo3::Py<pyo3::PyAny>>,
```

Comment:
Marking for review: I don't really know if `subject` should be added here, or not. The `subjects` field was made optional (and renamed but more on that later) since a custom policy might allow certs without a SAN extension, so it made sense to me to have some sort of subject field that's also present. But it's also true that most certificates will probably contain alternative names, and getting the subject from the certificate is straightforward.

Another thing is the naming - I feel like a rename was needed here since `subjects` and `subject` are just too similar, but I don't really like `sans` in the end. Maybe something like `alternative_names`? Suggestions welcome.

Reply:
Resolving, discussion moved to https://github.com/pyca/cryptography/pull/11559#discussion_r1754388083